src/App.js

```diff
@@ -73,6 +77,12 @@ function App() {
   function deleteCard(obj) {
     setLocalStorage(localStorage.filter((ware) => ware.id !== obj.id));
   }
+  function saveTheData(name, email, address) {
+    /* console.log(name, email, address); */
+    setInputData({ name: name, email: email, address: address });
+    console.log(inputData);
```

Comment:
same here, you can delete the console.logs in lines 81 and 83

Reply:
already done, thank you!